### PR TITLE
Add a way to start up a minimal ceph cluster

### PR DESCRIPTION
**Pull request by jtlayton**
There are very few differences between cluster.yaml and this file. yaml doesn't have an "include" directive though, so I think the duplication is unavoidable. Probably we could strip out some of the comments, but this does seem to do the right thing.

[skip ci]

### CODE REVIEW

File: cluster/examples/kubernetes/ceph/bootstrap.yaml [hunk 1]
```diff
@@ -0,0 +1,287 @@
+#
+# This file is intended to start a minimal ceph cluster consisting of just a
+# single mon and a mgr daemon. No actual storage is provisioned here. Once the
+# minimal cluster is set up using this yaml file, it's then possible to start
+# up the toolbox and expand the cluster using the command line (and eventually,
+# the dashboard).
+#
+# If one is deploying on a prisine k8s cluster, then start by picking an
```
Comment (mykaul) [inline cluster/examples/kubernetes/ceph/bootstrap.yaml]: pristine

Comment (jtlayton) [reply cluster/examples/kubernetes/ceph/bootstrap.yaml]: Thanks, fixed.

**Comment by jtlayton** [comment]
Ok. Happy to trim out more of it. Let me see how slim I can make it.

**Comment by leseb** [comment]
For reference, I was using this: https://github.com/leseb/openshift-ansible/blob/af25bb3bc4d262bdb1fbda04c849cea13bafd7cf/roles/openshift_storage_ceph/templates/cluster.yml.j2#L153-L178

**Comment by jtlayton** [comment]
Done. I trimmed off most of the last section and made sure it deployed the cluster correctly. FWIW, I'd like to see this merged soon as I'm working on an article that walks the reader through deploying a minimal ceph cluster and I'd like to be able to refer to this file.

**Comment by jtlayton** [comment]
Sure:

    No spec.storage section, and a spec.mon.count set to 1 instead of 3.

It sort of seems like a waste to duplicate so much for this, but such is yaml, apparently.

**Comment by jtlayton** [comment]
Added a couple of fixes for RBAC problems I hit while testing with minikube, with many thanks for @travisn for his help. We may want to make the same changes to cluster.yaml.

**Comment by jtlayton** [comment]
Done. I did leave a comment at the top about the purpose of the file, and some of the comments over the RBAC stuff, but I took out the rest.

**Comment by jtlayton** [comment]
One more patch for good measure. Let's fix the mode on the yaml files so that they aren't executable.

**Comment by travisn** [comment]
Looks good. I just opened #2700 with some related thoughts on creating new samples, if you have any suggestions.

**Pull request merged**